src/reaction_message.coffee

```diff
@@ -0,0 +1,17 @@
+{Message} = require.main.require 'hubot'
+
+class ReactionMessage extends Message
+   # Represents a message generated by an emoji reaction event
+   # - this was copied from the hubot-slack api and should function similarly
+   #
+   # type      - A String indicating 'reaction_added' or 'reaction_removed'
+   # user      - A User instance that reacted to the item.
+   # reaction  - A String identifying the emoji reaction.
+   # item_user - A String indicating the user that posted the item.
+   # item      - An Object identifying the target message, file, or comment item.
+   # event_ts  - A String of the reaction event timestamp.
+   constructor: (@type, @user, @reaction, @item_user, @item, @event_ts) ->
+       super @user
+       @type = @type.replace('reaction_', '')
```

Comment:
wouldn't we want to store the item, at the very least, in the reaction so that we can handle things like editing it, replying to it, or some such thing?

Reply:
We've had this issue before where storing the items from Discord.js breaks robot.brain because of the circular references.  As of now, I'm planning to re-fetch the items if needed again.

Reply:
right, but certain ones are small enough. I'm referring to at least the string itself of the mesage content.